packages/api/src/tasks/checkAlerts.ts

```diff
@@ -1,6 +1,7 @@
 // --------------------------------------------------------
 // -------------- EXECUTE EVERY MINUTE --------------------
 // --------------------------------------------------------
+import pMap, { pMapSkip } from '@esm2cjs/p-map';
```

Comment:
Not sure if there's a good alternative, but this hasn't been updated in 3 years (abandoned most likely)